gemfiles/rails_4.2.gemfile

```diff
@@ -4,9 +4,10 @@ source "https://rubygems.org"
 
 gem "minitest"
 gem "minitest-rg"
-gem "appraisal", "~> 0.5"
-gem "debugger", :platforms=>[:ruby_20]
+gem "appraisal"
+gem "debugger", platforms: [:ruby_20]
 gem "bump"
 gem "rake"
 gem "activesupport", "~> 4.2.11"
 
+gemspec path: "../"
```

Comment:
Does the new Appraisal add this line?

Reply:
It was removed in the commit prior and added back, I'll fix that it up.